doc/source/whatsnew/v0.23.0.txt

```diff
@@ -216,6 +216,7 @@ Deprecations
 - ``Series.valid`` is deprecated. Use :meth:`Series.dropna` instead (:issue:`18800`).
 - :func:`read_excel` has deprecated the ``skip_footer`` parameter. Use ``skipfooter`` instead (:issue:`18836`)
 - The ``is_copy`` attribute is deprecated and will be removed in a future version (:issue:`18801`).
+- The ``convert_datetime64`` parameter has been deprecated and the default value is now ``False`` in :func:`to_records` as the NumPy bug motivating this parameter has been resolved (:issue:`18160`).
```

Comment:
Should <code>:func:\`to_records\`</code> actually be <code>:func:\`DataFrame.to_records\`</code>?

Reply:
1) @jschendel : I believe you are correct

2) @reidy-p : This sentence is a run-on.  Break it up into two sentences (and add comma's at the conjunctions).